Cargo.toml

```diff
@@ -9,6 +9,7 @@ members = [
   "crates/test-macros",
   "crates/config",
   "crates/runner",
+  "fuzz",
```

Comment:
This is intentional. Added it outside of the `crates` directory, given the infrastructure nature of this crate.